### PR TITLE
fix: Set scaleType programmatically for floating action buttons

### LINKED ISSUE #675: Fab icon is not in center due to Sdk 28
**Describe the bug**
Icon on the floating action button is not is center.

**Screenshots**
<p align="center">
 
</p>

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |MI MAX 2|
|Android Version|Nougat 7.1 |

**Additional context**
This problem arises due to upgrading SDK version to 28.

**Would you like to work on the issue?**
- [x] Yes

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #675 

Screenshots for the change:
<p align="center">
 
 
 
</p>

### CODE REVIEW

File: app/src/main/res/layout/item_card_events.xml [hunk 1]
```diff
@@ -21,7 +21,6 @@
             android:layout_gravity="end"
             android:layout_marginTop="@dimen/fab_margin_top"
             android:layout_marginRight="80dp"
-            android:scaleType="center"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/item_card_events.xml]: Try app:scaleType

Comment (liveHarshit) [reply app/src/main/res/layout/item_card_events.xml]: It is showing error. `Android resource linking failed`

**Comment by iamareebjamal** [comment]
@liveHarshit Let the scaleType remain in XML so that when it is fixed. it is easy to revert

**Comment by iamareebjamal** [comment]
Also, create an issue to revert when it is fixed

**Comment by liveHarshit** [comment]
Updated

**Pull request merged**